### ISSUE TITLE
Add early stopping

### ISSUE THREAD

**Issue by Jayanth-Balasubramanian** [Jayanth-Balasubramanian/Pneumonia-Detection#2]
### Problem:
Currently the number of epochs is set (somewhat) arbitrarily to 10

### Solution:
Early stopping using validation accuracy as a metric (ie stop if validation accuracy does not improve even after X epochs) can stop training before overfitting
